pkg/database/plugin.go

```diff
@@ -199,6 +199,9 @@ type iOperationCollection interface {
 	// ResolveOperation - Resolve operation upon completion
 	ResolveOperation(ctx context.Context, id *fftypes.UUID, status fftypes.OpStatus, errorMsg string, output fftypes.JSONObject) (err error)
```

Comment:
Now we have a generic `UpdateOperation` - I wonder if it would be more consistent to remove the `ResolveOperation` from the DB layer as that's just a thin wrapper:
https://github.com/hyperledger/firefly/blob/60e00ff8041a81b5e038ec53f23a7a430f058a09/internal/database/sqlcommon/operation_sql.go#L178-L186

Reply:
Yea, was thinking the same thing. I can do this.

Reply:
After looking at this again, there are a lot of places where we invoke `ResolveOperation`, and this change would take all of those from 1 line to 5 lines. So it would be more consistent, but more verbose as well. I slightly prefer keeping the concise helper.

Could also move the helper somewhere other than the database layer. For instance, it could be on the Operations Manager (but would then need to add a dependency from Event Manager on Operations Manager).